Makefile

```diff
@@ -34,16 +37,26 @@ LDFLAGS=-ldflags "-X main.version=$(VERSION) -X main.commit=$(COMMIT) -X main.da
 # PID file will keep the process id of the server.
 PID_PROOF_MOCK := /tmp/.$(PROJECT_NAME).proof.pid
 
-# Make is verbose in Linux. Make it silent.
+# GNU make is verbose. Make it silent.
 MAKEFLAGS += --silent
 
+# For go 1.13-1.15 compatibility
+# https://maelvls.dev/go111module-everywhere/
+export GO111MODULE=on
```

Comment:
maybe it's better to add it to the `GOENVVARS` instead of export as global

Reply:
I had problems in two places
https://github.com/hermeznetwork/hermez-node/blob/9ff7ed78337e7a99b977be1b26ebc6239b188a78/Makefile#L93
https://github.com/hermeznetwork/hermez-node/blob/9ff7ed78337e7a99b977be1b26ebc6239b188a78/Makefile#L120
both of them do not use `GOENVVARS`